app/views/charges/_form.html.erb

```diff
@@ -3,7 +3,15 @@ When adding a charge, please ensure that you add a detailed description. This al
 <%= simple_form_for @charge, :html => { :class => 'form-horizontal' } do |f| %>
   <div class="form-inputs">
     <%= f.association :organization %>
-    <%= f.association :charge_type %>
+    <div class = "row">
+      <div class = "col-md-8" style = "padding-left:112px;">
+        <%= f.association :charge_type, :input_html=> { :onchange => 
+        raw("$.getJSON('/charge_types/'+"+"this.value"+",function(response){$('#charge_amount').val(response)});") } %>
+      </div>
+      <div class = "col-md-4" style = "padding-left:76px;">
```

Comment:
This should probably check if the user can create a new charge type and not display otherwise. Although it doesn't matter as long as the access to create charges and chargetypes is the same (which I think you should change as well).